sql/core/src/test/scala/org/apache/spark/sql/execution/streaming/state/RocksDBStateStoreSuite.scala

```diff
@@ -177,6 +185,33 @@ class RocksDBStateStoreSuite extends StateStoreSuiteBase[RocksDBStateStoreProvid
 
   override def getDefaultSQLConf(
     minDeltasForSnapshot: Int,
-    numOfVersToRetainInMemory: Int): SQLConf = new SQLConf()
+    numOfVersToRetainInMemory: Int): SQLConf = SQLConf.get
+
+
+  override def testQuietly(name: String)(f: => Unit): Unit = {
+    test(name) {
+      quietly {
+        f
+      }
+    }
+  }
+
+  override def testWithAllCodec(name: String)(func: => Any): Unit = {
+    codecsInShortName.foreach { codecShortName =>
+      super.test(s"$name - with codec $codecShortName") {
+        withSQLConf(SQLConf.STATE_STORE_COMPRESSION_CODEC.key -> codecShortName) {
+          func
+        }
+      }
+    }
+
+    CompressionCodec.ALL_COMPRESSION_CODECS.foreach { codecShortName =>
```

Comment:
I know you copied the code from existing test, but it seems like it's codec classname rather than short name. Shall we fix it at least here (and probably other can be dealt with MINOR PR afterwards)?

Reply:
Renamed to codecName